frontend/src/routes/userLand/children/gallery/+page.svelte

```diff
@@ -14,6 +14,7 @@ import { alertStore } from "$lib/stores/alertStore.svelte";
 import { currentChild } from "$lib/stores/childrenStore.svelte";
 import { type CardElement, type CardStyle } from "$lib/util";
 import { Heading, Spinner } from "flowbite-svelte";
+import { displayChildImages } from "../../../../features";
```

Comment:
Could we put the features.ts in lib and use an absolute import? Makes this import less brittle than a relative one if we refactor these files later
```suggestion
import { displayChildImages } from "$lib/features";
```